### PR TITLE
Add function to install openvswitch pkg

### PR DESCRIPTION
**Pull request by yalzhang**
As openvswitch package name varies on different OS version, add a function
to install it.

Signed-off-by: Yalan <yalzhang@redhat.com>

### CODE REVIEW

**Comment by yalzhang** [comment]
@chloerh  @chunfuwen  Please help to review, Thank you!

File: virttest/utils_net.py [hunk 1]
```diff
@@ -1269,6 +1263,24 @@ def clean_ovs_env(run_dir="/var/run/openvswitch", selinux_mode=None,
...
+    :return: True or False
+    """
+    if process.system("yum info openvswitch", ignore_status=True) == 0:
+        utils_package.package_install("openvswitch")
+    elif process.system("yum info openvswitch2.11", ignore_status=True) == 0:
+        utils_package.package_install("openvswitch2.11")
+    elif process.system("yum info openvswitch2.15", ignore_status=True) == 0:
+        utils_package.package_install("openvswitch2.15")
+    else:
+        logging.debug("Fail to install openvswtich package!")
```
Comment (Yingshun) [inline virttest/utils_net.py:1279]: :%s/logging.debug/logging.error

Comment (yalzhang) [reply virttest/utils_net.py]: Updated, Thank you!

File: virttest/utils_net.py [hunk 2]
```diff
@@ -1269,6 +1263,24 @@ def clean_ovs_env(run_dir="/var/run/openvswitch", selinux_mode=None,
...
+    Install openvswtich package
+
+    :return: True or False
+    """
+    if process.system("yum info openvswitch", ignore_status=True) == 0:
+        utils_package.package_install("openvswitch")
+    elif process.system("yum info openvswitch2.11", ignore_status=True) == 0:
+        utils_package.package_install("openvswitch2.11")
+    elif process.system("yum info openvswitch2.15", ignore_status=True) == 0:
+        utils_package.package_install("openvswitch2.15")
```
Comment (Yingshun) [inline virttest/utils_net.py:1277]: We can't cover all versions of openvswitch2 in this function, someone told me they may have different ovs packages at the same time and user would select the proper version before the testing.
```
openvswitch2.11-2.11.3-67.el8fdp.ppc64le : Open vSwitch
Repo        : FDP
Matched from:
Filename    : /usr/bin/ovs-vsctl

openvswitch2.12-2.12.0-23.el8fdp.ppc64le : Open vSwitch
Repo        : FDP
Matched from:
Filename    : /usr/bin/ovs-vsctl

openvswitch2.13-2.13.0-58.el8fdp.ppc64le : Open vSwitch
Repo        : FDP
Matched from:
Filename    : /usr/bin/ovs-vsctl
```
So I think there can be 2 options:
1. make the version configurable, for instance, ` install_openvswitch(ovs_ver="2.*")`
2. install the correct package in ci, and check if ovs-vsctl cmd exists in codes.
What do you think? @yalzhang

Comment (yalzhang) [reply virttest/utils_net.py:1277]: Thank you! Have add it to CI, it is cheaper, close this PR.

**Pull request closed without merging**